tests/codex/sales/testreservations.nim

```diff
@@ -254,6 +254,17 @@ asyncchecksuite "Reservations module":
     check updated.isErr
     check updated.error of BytesOutOfBoundsError
 
+  test "cannot release more bytes than size of reservation - concurrency test":
```

Comment:
Where's the concurrency in this test?

Reply:
Haha, yeah nowhere 🙈 That is what happens when you do not finish the work before the weekend... You forgot where you left 😅